src/widgets/accessibility-link-button/AccessibilityLinkButton.tsx

```diff
@@ -0,0 +1,50 @@
+import { WidgetBase } from '@dojo/framework/core/WidgetBase';
+import ThemedMixin, { theme, ThemedProperties } from '@dojo/framework/core/mixins/Themed';
+import { tsx } from '@dojo/framework/core/vdom';
+import { LinkProperties } from '@dojo/framework/routing/interfaces';
+import Link from '@dojo/framework/routing/Link';
+
+import * as css from './AccessibilityLinkButton.m.css';
+
+export interface AccessibiltyLinkButtonProperties extends ThemedProperties {
+	onClick?: () => void;
+	disabled?: boolean;
+	linkProperties?: LinkProperties;
+	href?: string;
+}
+
+@theme(css)
+export class AccessibilityLinkButton extends ThemedMixin(WidgetBase)<AccessibiltyLinkButtonProperties> {
+	protected render() {
+		const { key, onClick, disabled = false, linkProperties, href } = this.properties;
+
+		if (href) {
```

Comment:
I’m really not sure this makes a good widget, all the properties are optional but depending on which ones you provide the behaviour is overridden? For example if I pass `href` then both `linkProperties` and `onClick` no longer work.